lib/retriable/exponential_backoff.rb

```diff
@@ -25,19 +25,16 @@ def initialize(opts = {})
 
     def intervals
       intervals = Array.new(tries) do |iteration|
-        [base_interval * multiplier**iteration, max_interval].min
+        [base_interval * multiplier ** iteration, max_interval].min
       end
 
-      return intervals if rand_factor.zero?
```

Comment:
you already have that line... https://github.com/kamui/retriable/blob/1ca240edf2776b08df79f725c933de50adbfe5db/lib/retriable/exponential_backoff.rb#L39